src/component/LocaleContext.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const LocaleContext = React.createContext();
+
+export const withLocale = Component => props => (
+  <LocaleContext.Consumer>
+    {({locale}) => <Component {...props} locale={locale}/>}
+  </LocaleContext.Consumer>
+);
+
+export default LocaleContext;
```

Comment:
`react-intl` already has the locale in the [intlShape](https://github.com/yahoo/react-intl/wiki/API#intlshape). Do we need this?

Reply:
Yes, you're right. The `injectIntl` HOC provides locale via props: `this.props.intl.locale`. So, this custom solution is useless because our library is already coupled to the React-Intl.

Reply:
Removed from PR!